File: projects/m1/010-month-name-to-number-of-days/solutions/stefandifede/month-name-to-number-of-days.js
const userInput = prompt("insert a month")

const isThirty = ["april", "june", "september","november"]
const isThirtyone = ["january", "march", "may", "july", "august", "october", "december"]

function monthToNumber () {
    if (userInput == "february") {
        alert(userInput + " has 28 or 29 days")
    } else if (isThirty.includes(userInput)) {
        alert(userInput + " has 30 days")
    } else if (isThirtyone.includes(userInput)) {
        alert(userInput + " has 31 days")
    }
}

monthToNumber()

File: projects/m1/010-month-name-to-number-of-days/solutions/GiorgioCignitti/MonthNametoNumberofDays.py
month=input("insert a month ")
lowerCaseMonth=month.lower();

def checkMonth(month):
  if(month=="november" or month=="june" or month=="april" or month=="september" ):
    return f"30 days"
  elif(month=="february"):
    return  f"28 or 29 days"
  else:
    return f"31 days"

print(checkMonth(lowerCaseMonth))